packages/typespec-ts/test/unit/transform/transformSchemas.spec.ts

```diff
@@ -257,11 +268,22 @@ describe("#transformSchemas", () => {
       const property = first.properties![`"prop"`];
       assert.isNotNull(property);
       assert.deepEqual(property, {
-        type: '"English"|"Chinese"',
+        type: '"English" | "Chinese"',
         name: "string",
         typeName: "string",
-        description: "Possible values: English, Chinese",
-        enum: ["English", "Chinese"],
+        description: "Possible values: \"English\", \"Chinese\"",
+        enum: [
```

Comment:
Not sure this change would impact the mock sample gen or not?

Reply:
I don't think so, we have both ut and smoke-test for mock sample gen, and they are fine.  https://github.com/Azure/autorest.typescript/pull/2189/files#diff-e83f5c92c1dffc0b39a10e7bd9ecb3623b13e6be14273e3af761cd72390f139bL92-R92